### ISSUE TITLE
Cant delete run profiles

### ISSUE THREAD

**Issue by honzadolezil** [rgbkrk/atom-script#2297]

**Comment by tiz2202** [comment]
the same situation with me, does anyone know how to fix it

**Comment by Stormbender0025** [comment]
Yep I'm having the same problem, but I am trying to find where exactly these configurations are store so I can delete them manually but have had no luck till now

**Comment by Stormbender0025** [comment]
Update: I dont know what happened, but suddenly none of the previously created profiles are gone, like poof!
![image](https://user-images.githubusercontent.com/44270163/113551542-72879d00-9612-11eb-8bae-1f111566bd73.png)
